development/secrets-rotator/terraform/environments/dev/variables.tf

```diff
@@ -9,11 +9,17 @@ variable "application_name" {
   default     = "secrets-rotator"
 }
 
-variable "location" {
+variable "region" {
   type        = string
   description = "Google Cloud location to deploy the service account keys rotator to."
 }
```

Comment:
Here's also description that refers a `location` instead of `region`. If you could also adjust that it'd be great